### PR TITLE
[backend] separate USB from Hamilton backend

### PR DESCRIPTION
**Pull request by jkhales**
this allows USBBackend to be a more general class.

### CODE REVIEW

**Comment by rickwierenga** [comment]
foundational change that will make future backends easier to add!

can you move the usb backend file to the top level / out of plr.liquid_handling?

**Comment by jkhales** [comment]
> foundational change that will make future backends easier to add!
> 
> can you move the usb backend file to the top level / out of plr.liquid_handling?

good call. I consolidated it with the `machines.py` file into a top level `machines` directory.

File: pylabrobot/machines/resources.py [hunk 1]
```diff
@@ -23,19 +25,6 @@ async def wrapper(self: Machine, *args, **kwargs):
     return await func(self, *args, **kwargs)
   return wrapper
 
-
```
Comment (jkhales) [inline pylabrobot/machines/resources.py:26]: moved this to `backends.machine.py`

**Comment by rickwierenga** [comment]
I think machines/machine.py could be a bit more descriptive than machines/resource.py (so we only have one resource.py in the entire project / the 'main' class name then matches the file name)

**Comment by rickwierenga** [comment]
Thanks!

**Pull request merged**